src/Fixer/Phpdoc/PhpdocSeparationFixer.php

```diff
@@ -24,8 +27,30 @@
 /**
  * @author Graham Campbell <graham@alt-three.com>
  */
-final class PhpdocSeparationFixer extends AbstractFixer
+final class PhpdocSeparationFixer extends AbstractFixer implements ConfigurableFixerInterface
 {
+    /**
+     * Groups of tags that should be allowed to immediately follow each other.
+     *
+     * @var array
+     */
+    private static $defaultGroups = [
+        ['deprecated', 'link', 'see', 'since'],
+        ['author', 'copyright', 'license'],
+        ['category', 'package', 'subpackage'],
+        ['property', 'property-read', 'property-write'],
+    ];
+
+    public function configure(array $configuration = null)
+    {
+        if (null === $configuration || [] === $configuration) {
```

Comment:
Why `[]`? What if someone wants to separate all tags?

Reply:
Found the reason. It's because of https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/2.17/src/AbstractFixer.php#L121

> Passing NULL to set default configuration is deprecated and will not be supported in 3.0, use an empty array instead.

So, if you want to separate all tags, pass it separately